### ISSUE TITLE
[bug] unknown VS version in documentation

### ISSUE THREAD

**Issue by Johnnyxy** [conan-io/conan#5317]
Hi there,
in the documentation there is written a Visual Studio version number which does not exist:
https://docs.conan.io/en/latest/reference/conanfile/methods.html#self-info-vs-toolset-compatible-self-info-vs-toolset-incompatible

The table specifies a **Visual Studio Version** ```13``` for **Compatible toolset** ```v120```.
While the toolset exists the Visual Studio version does not.

**Comment by lasote** [comment]
You are right. I'm transferring the issue to the docs repo. I don't know from where it came from. Probably a typo
